contrib/test/ci/build/crun.yml

```diff
@@ -7,6 +7,26 @@
     force: "{{ force_clone | default(False) | bool}}"
     version: "HEAD"
 
+- name: Build go-md2man for RHEL9
+  block:
+    - name: clone kubernetes source repo
```

Comment:
the name of each of these still refererences kubernetes

Reply:
whoops, good catch